autoarray/plot/mat_wrap/include.py

```diff
@@ -80,16 +77,16 @@ def section(self):
 class Include2D(AbstractInclude):
     def __init__(
         self,
-        origin: typing.Optional[bool] = None,
-        mask: typing.Optional[bool] = None,
-        border: typing.Optional[bool] = None,
-        grid: typing.Optional[bool] = None,
-        mapper_data_pixelization_grid: typing.Optional[bool] = None,
-        mapper_source_pixelization_grid: typing.Optional[bool] = None,
-        mapper_source_grid_slim: typing.Optional[bool] = None,
-        parallel_overscan: typing.Optional[bool] = None,
-        serial_prescan: typing.Optional[bool] = None,
-        serial_overscan: typing.Optional[bool] = None,
+        origin: Optional[bool] = None,
+        mask: Optional[bool] = None,
+        border: Optional[bool] = None,

```

Comment:
Why not default to False?

Reply:
The default behaviour is that if it is None, it loads True or False from a config. It is only when a user inputs True or False that the config is bypassed.